src/button-text-card.ts

```diff
@@ -278,6 +288,7 @@ export class BoilerplateCard extends LitElement {
         box-shadow: 2px 2px rgba(0, 0, 0, 0);
         padding: 16px;
         outline: none;
+        cursor: pointer;
```

Comment:
Good idea but I think we should only add this when the user defined a `tap_action` or `entity`.